### PR TITLE
#107: product page gallery : update style fotorama from  Magento 2.1

### PR DESCRIPTION
**Pull request by talalus**

### CODE REVIEW

File: styles/blocks/_gallery.scss [hunk 1]
```diff
@@ -1,255 +1,24 @@
...
-        margin-left: 20px;
-        cursor: pointer;
-    }
-
-    .fotorama__zoom-out {
-        @extend .fotorama-sprite;
-        top: 51px;
-        background-position: 0 (-$size-fotorama-block);
-    }
+// /**
```
Comment (Igloczek) [inline styles/blocks/_gallery.scss]: Let's remove this useless Magento license comments ;)

File: styles/blocks/_gallery.scss [hunk 2]
```diff
@@ -1,255 +1,24 @@
...
-}
+@import 'gallery/variables';
+@import 'gallery/mixins';
+@import 'gallery/extends';
+@import 'gallery/focus';
+@import 'gallery/fullscreen';
 
 .fotorama__zoom-in,
 .fotorama__zoom-out {
     display: none;
```
Comment (Igloczek) [inline styles/blocks/_gallery.scss:9]: Can we move all styles below imports to partials too?

File: styles/blocks/_gallery.scss [hunk 3]
```diff
@@ -1,255 +1,24 @@
-$fotorama-duration-time: 0.3s;
```
Comment (Igloczek) [inline styles/blocks/_gallery.scss:1]: Would be great to move gallery styles to some module dir... But there is nothing like `MagentoGallery`, so probably best option will be `MagentoCatalog`.

**Review by Igloczek — CHANGES_REQUESTED** [review]

File: styles/blocks/_gallery.scss [hunk 4]
```diff
@@ -1202,32 +979,83 @@ $fotorama-thumb-arrow: 30px;
...
     .loader img {
         position: absolute;
     }
 }
 
-body.fotorama__fullscreen {
-    overflow-y: hidden;
+// Styles for spinner in gallery.
+.fotorama__spinner {
+    background-image: url('#{$baseDir}../images/loader-1.gif');
```
Comment (JustinElst) [inline styles/blocks/_gallery.scss]: I was getting a 404 for loader-1.gif. After updating this line it worked. I think this should be:
`background-image: url('#{$baseDir}images/loader-1.gif');`

Comment (Igloczek) [reply styles/blocks/_gallery.scss]: `$baseDir` AFAIK is just a `../`, so it should looks like `background-image: url('../mages/loader-1.gif');`

File: styles/blocks/_gallery.scss (hunk 2 above)
Comment (talalus) [reply styles/blocks/_gallery.scss:9]: Sure, but I followed default Magento LESS structure so it'll be easy later to track changes and update blank theme and child themes

File: Magento_Catalog/styles/module/_gallery.scss [hunk 5]
```diff
@@ -1,255 +1,19 @@
...
-        @extend .fotorama-sprite;
-        top: 0;
-        background-position: 0 0;
-    }
-}
+@import 'gallery/variables';
+@import 'gallery/mixins';
+@import 'gallery/extends';
+@import 'gallery/focus';
+@import 'gallery/fullscreen';
```
Comment (Igloczek) [inline Magento_Catalog/styles/module/_gallery.scss:6]: Can we move all of this code below to new partial or ideally few smaller partials (it's fu... 1000 lines of styles :O )?
This file should contain only imports.

**Review by Igloczek — CHANGES_REQUESTED** [review]

File: styles/blocks/_gallery.scss (hunk 2 above)
Comment (Igloczek) [reply styles/blocks/_gallery.scss:9]: If we copy paste everything to new file, it will be easy to track this changes.

**Comment by Igloczek** [comment]
Karol make all necessary changes in #110, so I merged this one instead of this changes.

**Pull request closed without merging**